Modules/Core/Common/test/itkPointGeometryTest.cxx

```diff
@@ -98,7 +98,7 @@ itkPointGeometryTest(int, char *[])
   std::cout << "Squared Euclidean distance between pc and pb = ";
   std::cout << distance2 << std::endl;
 
-  vnl_vector_ref<ValueType> vnlVector = pa.GetVnlVector();
+  auto vnlVector = pa.GetVnlVector();
```

Comment:
// This is fine as is,  const version would be slightly better.
const auto vnlVector = pa.GetVnlVector();